.github/dependabot.yml

```diff
@@ -5,3 +5,7 @@ updates:
   schedule:
     interval: weekly
   open-pull-requests-limit: 10
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
just wondering if you prefer `weekly` over `daily` since I see dependabot originally configured for weekly